action.yml

```diff
@@ -143,4 +146,4 @@ inputs:
 
 runs:
   using: 'node16'
```

Comment:
btw, shouldn't this be like `node20`?

Reply:
If things work with the newer version feel free to bump. I just pinned it previously to not deal with GitHub upgrading what ubuntu-latest has as a default.